templates/spin_sidebar.hbs

```diff
@@ -106,16 +106,14 @@
                     Tutorials
                 </label>
                 <ul class="menu-list accordion-menu-item-content">
-                    <li><a {{#if (active_content request.spin-path-info "/spin/serverless-ai-tutorial" )}} class="active"
-                            {{/if}} href="{{site.info.base_url}}/spin/serverless-ai-tutorial">Build a Sentiment Analysis API Using Serverless AI Inferencing</a></li>
-                    <li><a {{#if (active_content request.spin-path-info "/spin/url-shortener" )}} class="active" {{/if}}
-                            href="{{site.info.base_url}}/spin/url-shortener">Building a URL Shortener With Spin</a></li>
-                    <li><a {{#if (active_content request.spin-path-info "/spin/spin-oci" )}} class="active" {{/if}}
-                            href="{{site.info.base_url}}/spin/spin-oci">Spin Apps in Registries</a></li>
-                    <li><a {{#if (active_content request.spin-path-info "/spin/kv-store-tutorial" )}} class="active"
-                            {{/if}} href="{{site.info.base_url}}/spin/kv-store-tutorial">Persistent Data Locally With
-                            Key Value
-                            Store</a></li>
+                    <li><a {{#if (active_content request.spin-path-info "/spin/ai-sentiment-analysis-api-tutorial" )}} class="active"
+                            {{/if}} href="{{site.info.base_url}}/spin/ai-sentiment-analysis-api-tutorial">AI Sentiment Analysis API Tutorial</a></li>
```

Comment:
I question whether we need to say "Tutorial" in the TOC when this is already in the Tutorials section?

Reply:
Yeah, good point. 
I will remove the word “Tutorial” from the visible/clickable menu text.

Reply:
I also think the same goes for the fact that the user is on the “Spin” project; and each page has the `/spin` location in the URLs path (which is why I removed any `Spin` prefixes from the visible/clickable menu text).

Reply:
Yes, good call.  I wonder if the registry one would then go to "Storing Applications in Registries" and the URL shortener one to something like "Building a URL Shortener" perhaps?  I think they need that bit of extra context but agree we can drop the Spin part.